gcloud/bigquery/job.py

```diff
@@ -658,91 +764,123 @@ def _build_resource(self):
-        if 'creationTime' in statistics:
-            statistics['creationTime'] = float(statistics['creationTime'])
-        if 'startTime' in statistics:
-            statistics['startTime'] = float(statistics['startTime'])
-        if 'endTime' in statistics:
-            statistics['endTime'] = float(statistics['endTime'])
 
-        self._properties.update(cleaned)
+class _CopyConfiguration(object):
+    """User-settable configuration options for copy jobs."""
+    # None -> use server default.
+    _create_disposition = None
+    _write_disposition = None
 
-    def begin(self, client=None):
-        """API call:  begin the job via a POST request
 
-        See:
-        https://cloud.google.com/bigquery/docs/reference/v2/jobs/insert
+class CopyJob(_BaseJob):
+    """Asynchronous job: copy data into a BQ table from other tables.
 
-        :type client: :class:`gcloud.bigquery.client.Client` or ``NoneType``
-        :param client: the client to use.  If not passed, falls back to the
-                       ``client`` stored on the current dataset.
-        """
-        client = self._require_client(client)
-        path = '/projects/%s/jobs' % (self.project,)
-        api_response = client.connection.api_request(
-            method='POST', path=path, data=self._build_resource())
-        self._set_properties(api_response)
+    :type name: string
+    :param name: the name of the job
 
-    def exists(self, client=None):
-        """API call:  test for the existence of the job via a GET request
+    :type destination: :class:`gcloud.bigquery.table.Table`
+    :param destination: Table into which data is to be loaded.
 
-        See
-        https://cloud.google.com/bigquery/docs/reference/v2/jobs/get
+    :type sources: list of :class:`gcloud.bigquery.table.Table`
+    :param sources: Table into which data is to be loaded.
 
-        :type client: :class:`gcloud.bigquery.client.Client` or ``NoneType``
-        :param client: the client to use.  If not passed, falls back to the
-                       ``client`` stored on the current dataset.
-        """
-        client = self._require_client(client)
+    :type client: :class:`gcloud.bigquery.client.Client`
+    :param client: A client which holds credentials and project configuration
+                   for the dataset (which requires a project).
+    """
+    def __init__(self, name, destination, sources, client):
+        super(CopyJob, self).__init__(name, client)
+        self.destination = destination
+        self.sources = sources
+        self._configuration = _CopyConfiguration()
 
-        try:
-            client.connection.api_request(method='GET', path=self.path,
-                                          query_params={'fields': 'id'})
-        except NotFound:
-            return False
-        else:
-            return True
+    @property
+    def create_disposition(self):
+        """Handling for missing destination table.
```

Comment:
I don't know what this means or how it relates to `create_disposition`. Can you explain? (And then maybe we'll need to change the docstring.)